### PR TITLE
user2:adding user access scenario

### PR DESCRIPTION
**Pull request by gitcliff**
link:https://github.com/IsantePlus/isanteplus-qaframework/issues/23

### CODE REVIEW

**Comment by gitcliff** [comment]
cc @mozzy11

File: src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/UserManagementSteps.java [hunk 1]
```diff
@@ -161,4 +161,33 @@ public void retireMessage() throws Exception {
...
+    @And("Select the modifier icon in the top right corner of the ‘details du compte d’utilisateur’")
+    public void selectModifierIcon() throws Exception {
+        accountsPage.clickEditButton();
+    }
+
+    @And("select the appropriate privileges under ‘capacites’ checkboxes {string} {string}")
+    public void selectAppropriatePrivillages(String userName, String privillage) throws Exception {
+        accountsPage.enterUserName(userName);
+        accountsPage.selectPrevillageLevel(privillage);
+        Thread.sleep(5000);
```
Comment (sherrif10) [inline src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/UserManagementSteps.java:178]: These thread sleeps are not good to use , we better use selenium time waits utils

Comment (mozzy11) [reply src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/UserManagementSteps.java:178]: Thats true @sherrif10 .
The selenium wait polls the DOM based on a given condition , while the Thread.sleep ,sleeps the script without condition .
We shall refactor these out

Comment (gitcliff) [reply src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/UserManagementSteps.java:178]: thanks guys for the reviews

**Pull request merged**